manifests/o/OO-Software/ShutUp10/1.9.1435/OO-Software.ShutUp10.installer.yaml

```diff
@@ -12,9 +11,6 @@ Installers:
   InstallerUrl: https://dl5.oo-software.com/files/ooshutup10/OOSU10.exe
   InstallerSha256: 22D3A45792B749E70B908088E95C19ABAE0707B248FCB83744B23BC6F662425B
   ReleaseDate: 2022-06-02
-  AppsAndFeaturesEntries:
-  - DisplayName: O&O ShutUp10++
-    DisplayVersion: 1.9.1431
-    Publisher: O&O Software
+  Publisher: O&O Software

```

Comment:
```suggestion
```
Publisher field is not defined here. Passing it through `winget validate --manifest <path_to_manifest>`
```
Manifest Warning: Unknown field. [Publisher]
```